selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -937,7 +937,7 @@ public void selectAutocompleteProposal(String item) {
    */
   public void selectCompositeAutocompleteProposal(String item) {
     seleniumWebDriverHelper.waitAndClick(
-        By.xpath(format(AUTOCOMPLETE_CONTAINER + "/li/span[.='%s']", item)));
+        By.xpath(format(AUTOCOMPLETE_CONTAINER + "/li/span[text()='%s']", item)));
```

Comment:
just reuse this method where you need
``` 
public void selectAutocompleteProposal(String item) {
    seleniumWebDriverHelper.waitAndClick(
        By.xpath(format(AUTOCOMPLETE_CONTAINER + "/li/span[text()='%s']", item)));
  }
```
https://github.com/eclipse/che/blob/0b2ea498eb6662d50e81c2bbb4b5a84a058e22ec/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java#L927

Reply:
I'm pretty sure you're going to break a couple of tests in the java autocomplete tests with this test. Why would including the text of enclosed spans break any test?